basic-features.md

```diff
@@ -10,166 +10,166 @@ This document lists the features supported in each TiDB version. Note that suppo
-| `UNION`, `UNION ALL`                                                                                     | Y            | Y            | Y            | Y            |
-| [`EXCEPT` and `INTERSECT` operators](/functions-and-operators/set-operators.md)                          | Y            | Y            | Y            | N            |
-| `GROUP BY`, `ORDER BY`                                                                                   | Y            | Y            | Y            | Y            |
-| [Window Functions](/functions-and-operators/window-functions.md)                                         | Y            | Y            | Y            | Y            |
-| [Common Table Expressions (CTE)](/sql-statements/sql-statement-with.md)                                  | Y            | Y            | N            | N            |
-| `START TRANSACTION`, `COMMIT`, `ROLLBACK`                                                                | Y            | Y            | Y            | Y            |
-| [`EXPLAIN`](/sql-statements/sql-statement-explain.md)                                                    | Y            | Y            | Y            | Y            |
-| [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md)                                    | Y            | Y            | Y            | Y            |
-| [User-defined variables](/user-defined-variables.md)                                                     | Experimental | Experimental | Experimental | Experimental |
+| **SQL statements** [^2]                                                                                  | **5.3**      | **5.2**      | **5.1**      | **5.0**      | **4.0**      |
+|----------------------------------------------------------------------------------------------------------|:------------:|:------------:|:------------:|:------------:|:------------:|
+| Basic `SELECT`, `INSERT`, `UPDATE`, `DELETE`, `REPLACE`                                                  | Y            | Y            | Y            | Y            | Y            |
+| `INSERT ON DUPLICATE KEY UPDATE`                                                                         | Y            | Y            | Y            | Y            | Y            |
+| `LOAD DATA INFILE`                                                                                       | Y            | Y            | Y            | Y            | Y            |
+| `SELECT INTO OUTFILE`                                                                                    | Y            | Y            | Y            | Y            | Y            |
+| `INNER JOIN`, `LEFT\|RIGHT [OUTER] JOIN`                                                                 |RIGHT [OUTER] JOIN`                                                                 | Y            | Y            | Y            | Y            |
```

Comment:
There is a problem in the format for this -- it repeats `RIGHT [OUTER] JOIN`

Reply:
```suggestion
| `INNER JOIN`, `LEFT\|RIGHT [OUTER] JOIN`                                                                 | Y                                                                 | Y            | Y            | Y            | Y            |
```